mod/zaudio/views/default/js/zaudio/boot.js

```diff
@@ -0,0 +1,16 @@
+define(function (require) {
+	var $ = require('jquery');
+
+	return {
+		addBehavior: function (context) {
+			var $targets = $('[data-zaudio-player]', context);
+			if ($targets.length) {
+				require(['elgg/zaudio'], function (zaudio) {
```

Comment:
I was a bit surprised this works (async require inside definition) but it works fine, allowing the optional loading of that set of dependencies only if we need it. In this PR I do the same for CKeditor.